content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -130,11 +130,29 @@ Finally, run the following commands:
 
 ### APM and DogStatsD
 
-To configure DogstatsD clients and APM tracers automatically, inject the environment variables `DD_AGENT_HOST` and `DD_ENTITY_ID` by using one of the following:
+To configure and inject APM tracers libraries and DogstatsD clients automatically:
 
 - Add the label `admission.datadoghq.com/enabled: "true"` to your pod.
 - Configure the Cluster Agent admission controller by setting `mutateUnlabelled` (or `DD_ADMISSION_CONTROLLER_MUTATE_UNLABELLED`, depending on your configuration method) to `true`.
 
+To opt-in your container for library injection, use Pod annotations inside your application's YAML file to specify which language tracers and should be injected, along with their versions.
+
+The annotations are a `key: value` pair in the following format:
+
+```yaml
+admission.datadoghq.com/<language>-tracer.version: <tracer-version>
```

Comment:
your original draft didn't have the word `admission` in this yaml line. should it be there?